operator/v1/types_csi_cluster_driver.go

```diff
@@ -279,8 +279,35 @@ type VSphereCSIDriverConfigSpec struct {
 	// If cluster Infrastructure object has a topology, values specified in
 	// Infrastructure object will be used and modifications to topologyCategories
 	// will be rejected.
+	// +listType=set
 	// +optional
 	TopologyCategories []string `json:"topologyCategories,omitempty"`
+
+	// globalMaxSnapshotsPerBlockVolume is a global configuration parameter that applies to volumes on all kinds of
+	// datastores. If unset it defaults to 3.
```

Comment:
Can we do something like

```
If omitted, the platform chooses a default, which is subject to change over time, currently that default is 3.
```

This matches other fields.  Rather than promising a value.

Reply:
Sounds good, changed.